client/js/components/CreateGameDialog.vue

```diff
@@ -87,6 +97,17 @@ export default {
       this.$emit('error', err);
       this.cancelCreateGame();
     },
+    setRankedPreference(prefersRanked) {
+      try {
+        localStorage.setItem(LS_PREFERS_RANKED_NAME, prefersRanked === true);
+      } catch (err) {
+        // Local storage is full or something else happened so don't set the value
+      }
+      this.isRanked = prefersRanked;
+    },
+    getRankedPreference() {
+      return localStorage.getItem(LS_PREFERS_RANKED_NAME) === 'true';
+    },
```

Comment:
Woo venturing into new lands here. It begs questions around future features and when to store data in the db vs local storage etc but I don't think we need to establish a comprehensive plan for that now.

Reply:
We should create `localStorage` utilities once we start using this in more places, but ideally we'll add them to a user profile object. Maybe LS as a fallback for guests if we ever go that route.